docs/docs/installation/index.md

```diff
@@ -0,0 +1,44 @@
+# Install Mathesar
+
+This page contains instructions to install Mathesar on various platforms.
+
+## Guided installation using our install script
+- Use our install script to guide you install Mathesar via a series of prompts:
+    ```sh
+    bash <(curl -sfSL https://raw.githubusercontent.com/centerofci/mathesar/0.1.1/install.sh)
+    ```
+- The script sets up Mathesar using Docker Compose and configures the admin user [under the hood](./guided-install/under-the-hood.md).
+- !!! warning "Limitations"
+    This is a convenient way to install Mathesar. However, it requires `sudo` privileges (admin access), and sets up a limited set of configuration options. Use the [Docker Compose installation](./docker-compose/index.md) option to exert more control.
+- [Read detailed instructions](./guided-install/index.md)
```

Comment:
This is kind of dangling in the breeze. The 'limitations' box breaks up the section a lot. Consider changing the link text to something like 'Detailed Guided Installation Instructions'. The 'read' is implied by clicking the hyperlink.

Reply:
This commit https://github.com/centerofci/mathesar/commit/036f1745cc11277ec055740ab8a91505ccdf4c55 moves the limitations box below the link which improves the readability of the section.

The text is also modified.